frontend/src/components/AvailabilityFilter.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import FilterButton from "./FilterButton";
+import { useFilteredConsultants } from "@/hooks/useFilteredConsultants";
+
+export default function AvailabilityFilter() {
+  const { availabilityFilterOn, toggleAvailabilityFilter } =
+    useFilteredConsultants();
+
+  return (
+    <div>
```

Comment:
Lite tips: Det er fint å prøve å la DOMen være så flat som mulig, samt å generelt unngå div om man kan. Her kan vi fjerne den ene diven siden de er doble. Om du trenger den for layout, kan du få en litt penere struktur om du heller styler diven under så ting blir rett.

Reply:
Helt enig! 👍